docs/pages/management/operations/ca-rotation.mdx

```diff
@@ -1,261 +1,462 @@
 following order:
 
-- `standby` -> `init` -> `update_clients` -> `update_servers` -> `standby`
-
-Administrators can abort the rotation and revert all changes any time before
-the rotation is completed by entering the `rollback` phase.
-
-```code
-$ tctl auth rotate --manual --type=<Var name="type" description="Certificate authority to rotate"/> --phase=rollback 
-```
+1. `standby`: No rotation in progress. No operations have begun.
+1. `init`: A new certificate authority is issued, but not used.
+1. `update_clients`: The Teleport Auth Service uses the new CA to sign
+   certificates but continues to trust certificates signed by the original CA.
+1. `update_servers`: Teleport servers (Agents, Auth Service, and Proxy Service
+   instances) reload and start serving TLS and SSH certificates signed by the
+   new certificate authority, but still accept certificates issued by the
+   original certificate authority. This only applies to the Teleport [`host`
+   CA](#host).
+1. `standby`: No rotation in progress. All operations have completed.
+
+Before the final `standby` phase, you an also put the rotation in the `rollback`
+phase, aborting the rotation and returning to the original certificate
+authority.
 
-For example, if an admin has detected that some nodes failed to upgrade during
-`update_servers`, they can roll back to the previous certificate authority, and
-the phase transitions look like this:
+CA rotations can be **manual** or **semi-automatic**. In manual mode, admins
+must instruct the Teleport Auth Service to advance from one phase to the next.
+Between phases, admins can prepare their infrastructure to adjust to each
+change. In semi-automatic mode, the Teleport Auth Service cycles through each
+phase automatically, with a grace period between each phase.
 
-- `update_servers` -> `rollback` -> `standby`.
+## Prerequisites
 
-<Admonition>
-  Try rotation/rollback in manual mode first to understand all the edge-cases
-  and gotchas before going with semi-automatic version.
-</Admonition>
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
 
-To specify which certificate authority to rotate, you must provide a value via
-the `--type` flag. If no value is provided, `tctl` will display an error and exit.
+- (!docs/pages/includes/tctl.mdx!)
 
-## Manual rotation
+## Step 1/4. Choose a CA to rotate
 
-In manual mode, we manually transition between phases while monitoring the state
-of the cluster.
+When rotating a CA, you need to check that any infrastructure that relies on the
+CA has not lost connectivity. You may also need to export the new CA to your
+infrastructure. Choose one of the CAs below to determine how to keep it up to
+date during the migration.
 
-**Start the rotation**
+We recommend rotating a single CA at a time in order to reduce complexity. The
+exception is rotating the `db` and `db_client` CAs. 
 
-Initiate the manual rotation of host certificate authorities:
+|CA type|Certificate subjects|Verified by|
+|---|---|
```

Comment:
table rendering fix:
```suggestion
|---|---|---|
```

Reply:
I think we should drop the "verified by" column.

Accurately describing and enumerating the things that verify certs issued by a given CA is too much info.